docs/rfcs/RFC-2023-07-19-coop-content-naming.md

```diff
@@ -0,0 +1,56 @@
+
+> July 19, 2023
+# RFC on the name "Cooperative Content"
+
+A last-call before publishing any packages to get some feedback on how the current name is
+interpreted.  The purpose of getting feedback is to check if there is some nebulous reason that this
+name would be counter-productive to its goal; not to determine a new name.
```

Comment:
Your perspective is so consistently different from mine which makes your input extra valuable so I'm trying to imagine what you are seeing.  I agree that the "content" could be many forms of information.  My thinking was that "content" and "information" are synonymous but perhaps that is not the common understanding.

Regarding the "goal", yes I think this is about managing/enforcing the relationships between agents and a common goal.  The "goal" is embodied as a group entry, and the relationships would be the contributors.  Since the "goal" is conceptual, it takes human judgment to decide which contributors are being cooperative.  That judgment is currently made by the "admin" role, but more strategies for group evolution can be added later.  I like the way you worded it with "protection from people who aren't cooperating"; which seems clearer than the descriptions I wrote.